tools/sgx-capability/check-sgx-capability.c

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020 NEC Solution Innovators, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright (C) 2011-2019 Intel Corporation. All rights reserved.
```

Comment:
Maybe also mention where the BSD 3-Clause code comes from?

Reply:
Ok, I'll mention it.